### ISSUE TITLE
Text overlaps with long routes

### ISSUE THREAD

**Issue by chimon2000** [desirable-objects/hapi-ending#8]
When defining a long route, the text of the route header overlaps with the examples.  It would be nice if the styles were more adaptive.

![image](https://user-images.githubusercontent.com/6907797/47129874-26d7b200-d265-11e8-907b-cd49376dcbe9.png)

**Comment by antony** [comment]
Agreed. I'm rewriting the frontend in #7 so I will address this issue as part of that work :)
